tools/manifest/tests/test_sourcefile.py

```diff
@@ -642,6 +644,29 @@ def test_relative_testdriver(ext):
     assert not s.has_testdriver
 
 
+@pytest.mark.parametrize("ext", ["htm", "html"])
```

Comment:
Nit; there are no inverse tests here, so an implementation of `quic` that just returned true would pass.

Reply:
Good point! Done.